vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -2024,3 +2075,30 @@ func NewOvfHelperParamsFromVMResource(d *schema.ResourceData) *ovfdeploy.OvfHelp
 	}
 	return ovfParams
 }
+
+func configureVTPM(ctx context.Context, vm *object.VirtualMachine) error {
```

Comment:
This function (`configureVTPM`) doesn't seem to be used. Letfover when attempting to perform a removal operation to for the device in the resource?

Reply:
correct did not catch that